README.md

```diff
@@ -6,22 +6,52 @@
 
 [![codecov.io](http://codecov.io/github/ahnlabb/BioformatsLoader.jl/coverage.svg?branch=master)](http://codecov.io/github/ahnlabb/BioformatsLoader.jl?branch=master)
 
-## Usage
+## Dependencies
 
-Requires `bioformats_package.jar`
+Depends on `bioformats_package.jar`
+
+## Install
+
+```julia
+Pkg.add("https://github.com/ahnlabb/BioformatsLoader.jl") # Change if forked
+Pkg.build("BuildformatsLoader") # Downloads bioformats_package.jar and ome.xsd
```

Comment:
Shouldn't be necessary, `add` performs a `build`.

Reply:
Great. I will take out the build step or maybe just comment it out.

I am wondering if it makes sense to distribute ome.xsd with this package.

Reply:
It might make sense and since ome.xsd is CC BY 3.0 we definitely could.

Reply:
Omitted the build step, but added a description of what it does